spec/shared_behaviors/taggable.rb

```diff
@@ -14,7 +14,7 @@ def tag_link(s)
 
   describe '.format_tags' do
     before do
-      @str = '#what #hey #vöglein'
+      @str = '#what #hey #vöglein #ինչո՞ւ #այո՜ո #սեւ֊սպիտակ #գժանո՛ց #മലയാണ്മ'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.